tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -280,4 +281,14 @@ then
   ${SETARCH_CMD} "${PYTHON}" src/python/grpcio_admin/setup.py \
       sdist bdist_wheel
   cp -r src/python/grpcio_admin/dist/* "$ARTIFACT_DIR"
+
+  # Build grpcio_observability source distribution
+  # Skips MacOS since grpcio_observability does not support MacOS.
+  if [ "$GRPC_UNIVERSAL2_REPAIR" == "" ]; then
```

Comment:
Having an API test [like this](https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests/unit/_api_test.py#L26) should be a blocker for publishing artifacts. We want to make sure there are no backward-incompatible changes accidentally made.

Reply:
I mean that we need to have a test that will fail if we remove a symbol from the API. That test will likely not live in the same file.